### PR TITLE
feat: [AP-65] - emoji chat render

### PR DESCRIPTION
**Pull request by KemoPaw**
**What this PR does** 📖
Have emojis not render as tiny as they were previously

**Which issue(s) this PR fixes** 🔨
https://satellite-im.atlassian.net/browse/AP-65

Fixes #

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

**Review by stavares843 — COMMENTED** [review]
@KemoPaw the emojis are awesome! 🎉 

Captura de ecrã 2021-12-14, às 00 09 34

**Review by stavares843 — COMMENTED** [review]
screenshot with a couple variations @WanderingHogan 

Captura de ecrã 2021-12-14, às 00 10 55

**Review by stavares843 — APPROVED** [review]

File: components/tailored/core/chatbar/Chatbar.vue [hunk 1]
```diff
@@ -245,13 +246,50 @@ export default Vue.extend({
...
+            let wrapAround = `<span class="mediumEmoji">${emojiFound}</span>`
+            newValue += wrapAround
+            i++
+          } else {
+            newValue += firstChar
+          }
+        }
+        return newValue
+      }
+      if (!newInput) {
```
Comment (WanderingHogan) [inline components/tailored/core/chatbar/Chatbar.vue:280]: don't need this if, the return statement in the if above will kick you out of the method before you get here

File: components/tailored/core/chatbar/Chatbar.vue [hunk 2]
```diff
@@ -245,13 +246,50 @@ export default Vue.extend({
...
+            /[\p{Emoji}\u200d]+/gu,
+          )
+            ? firstChar + secondChar
+            : false
+
+          if (emojiFound) {
+            let wrapAround = `<span class="mediumEmoji">${emojiFound}</span>`
+            newValue += wrapAround
+            i++
+          } else {
```
Comment (WanderingHogan) [inline components/tailored/core/chatbar/Chatbar.vue:274]: never else, use a guard clause here

**Review by WanderingHogan — CHANGES_REQUESTED** [review]
couple changes in comments

**Review by josephmcg — CHANGES_REQUESTED** [review]
https://jsfiddle.net/qu43byLn/7/
I was talking with Matt about this and he thought we should check the input string in a different way. The for loop may cause performance issues down the line. See if you can draw inspiration from the jsfiddle

You can also add regex rules to the config file at:
Core-PWA/config.ts

**Comment by KemoPaw** [comment]
https://github.com/Satellite-im/Core-PWA/pull/649

**Pull request closed without merging**